src/bridge/cockpitrouter.c

```diff
@@ -1101,18 +1114,16 @@ cockpit_router_add_bridge (CockpitRouter *self,
     match = NULL;
 
   /* See if we have any variables in the JSON */
```

Comment:
This comment should be updated, I think. (Or removed, code is much clearer now.)

Reply:
Done.  I actually expanded the comment a bit.  More explanation never hurts.